readme.md

```diff
@@ -5,8 +5,37 @@
 
 A CLI tool for managing a plethora of scheduled task python scripts.
 
+### Usage
+The work that forklift does is defined by plugins called [Pallets](src/forklift/models.py). `Pallet` is a base class that allows the user to define a job for forklift to perform by creating a new class that inherits from `Pallet`. Each pallet can have zero or more [Crates](src/forklift/models.py). A `Crate` is an class that defines data that needs to be moved from one location to another (reprojecting to web mercator by default). Crates are created by calling the `add_crates` (or `add_crate`) methods within `__init__` on the pallet. For example:
+```python
+class StringCratePallet(Pallet):
+    def __init__(self):
+        #: this call is required so that default properties are initialized
+        super(StringCratePallet, self).__init__()
+
+        destination_workspace = r'C:\\MapData'
+        source_workspace = path.join(data_folder, 'agrc@sgid10.sde')
+
+        self.add_crate('Counties', {'source_workspace': source_workspace,
+                                    'destination_workspace': destination_workspace})
+```
+For details on all of the members of the `Pallet` and `Crate` classes see [models.py](src/forklift/models.py).
+
+For examples of pallets see [samples/PalletSamples.py](samples/PalletSamples.py).
+
+#### CLI
+Interacting with forklift is done via the [command line interface](src/forklift/cli.py). Run `forklift -h` for a list of all of the available commands.
+
+#### Config File Properties
+`config.json` is created in the working directory after running `forklift config init`. It contains the following properties:
+- `warehouse` The folder where all of the repositories will be cloned into and where forklift will scan for pallets to run.
+- `sendEmails` Determines whether or not to send any emails. Set to `false` when testing.
```

Comment:
- forklift config set --key sendEmails --value False

might be nice to have this example there.